### DISCUSSION TITLE
Hue motion outdoor sensor (9290030674) - possibilities for updating firmware

### DISCUSSION THREAD

**Discussion by swinster** [Koenkk/zigbee2mqtt#15409] [Devices]
Hi,

Firstly, what is the latest firmware for this sensor? Zigbee2MQTT reports this as 2.53.6, but this document (https://www.philips-hue.com/en-gb/support/release-notes/accessories) suggests that in 2019 (!), the firmware revision was 6.1.1.27575. I'm not sure if the two version numbers are for the same thing but wondered what others saw as reported.

What are the possibilities for updating the firmware of the Hue motion outdoor sensor (https://www.zigbee2mqtt.io/devices/9290030674.html#philips-9290030674), assuming there is no Philips hue hub to connect it to temporarily, to perform the update? Zigbee2MQTT fails to update the firmware stating that it cannot be found:

```
 2022-12-07 00:48:22Checking if update available for 'Lodge - Philips Hue Outdoor Sensor'
Info 2022-12-07 00:48:24MQTT publish: topic 'zigbee2mqtt/bridge/response/device/ota_update/check', payload '{"data":{"id":"Lodge - Philips Hue Outdoor Sensor"},"error":"Failed to check if update available for 'Lodge - Philips Hue Outdoor Sensor' (No image available for imageType '283')","status":"error","transaction":"vidx6-1"}'
Error 2022-12-07 00:48:24Failed to check if update available for 'Lodge - Philips Hue Outdoor Sensor' (No image available for imageType '283')
```

From what I can understand from the OTA update documentation, not all Hue firmware is published, so I guess this is the issue here. Is there any other way to update the firmware of these devices?

Cheeers

**Comment by linkedupbits** [comment]
Using the Hue bridge/Hue app, an outdoor motion sensor (SML004) shows 2.53.6 and is “up to date”.
Hope that helps.

**Reply by swinster** [reply]
Perfect - thanks. 

Whether this will change in the future, I do not know, but for now, that is great.
